Documentation/cmdref/cilium-agent.md

```diff
@@ -58,6 +58,7 @@ cilium-agent [flags]
       --enable-endpoint-health-checking               Enable connectivity health checking between virtual endpoints (default true)
       --enable-endpoint-routes                        Use per endpoint routes instead of routing via cilium_host
       --enable-external-ips                           Enable k8s service externalIPs feature (requires enabling enable-node-port) (default true)
+      --enable-health-check-nodeport                  Enables a service health server running inside cilium-agent when running the NodePort services (default true)
```

Comment:
My reading of the description is that it enables the check for **all** nodeport svc. Maybe `Enables a healthcheck nodePort server for NodePort services with "healthCheckNodePort" being set` is a bit more accurate?